### ISSUE TITLE
e2e testing failed without breaking the ci

### ISSUE THREAD

**Issue by LinuxSuRen** [halo-dev/halo#6096]
### System information

As I mentioned, the e2e exit code is not be handled correctly.

### What is the project operation method?

Docker

### What happened?

![image](https://github.com/halo-dev/halo/assets/1450685/748d1a2d-d1b2-4073-b0c4-80208bb0f36d)

### Relevant log output

```shell
See https://github.com/halo-dev/halo/actions/runs/9513923185/job/26225382053?pr=6081
```

**Comment by JohnNiang** [comment]
Hi @LinuxSuRen , thank you for reporting the error!

I'm preparing to investigate the specific reason in depth.

**Comment by JohnNiang** [comment]
After my investigation, the problem may not be related to Halo.

I found the reverse test was introduced by <https://github.com/LinuxSuRen/api-testing/pull/317> and published into v0.0.16. And we upgraded api-testing in <https://github.com/halo-dev/halo/pull/5768>. I'm not sure if we need to adapt the reverse test.

**Comment by LinuxSuRen** [comment]
It seems `Halo` can take the auth information from cookies. And `atest` will keep the cookies among those test cases.
